src/components/new-conversation-dialog/groups/index.js

```diff
@@ -26,7 +26,7 @@ const Groups = ({
   onChangeGroupsQuery,
   onSearchGroups,
   hideNewConversation,
-  showCreateGroup,
+  flipCreateGroup,
```

Comment:
thought it would be easier to understand this code as `flipCreateGroup` executed on component load and can be both true and false, depends on organization settings. But now when opened code, wasn't so clear for me, so decided to rename it back to `showCreateGroup`, and also renamed type to `SHOW_CREATE_GROUP`